src/test/addrman_tests.cpp

```diff
@@ -1044,5 +950,35 @@ BOOST_AUTO_TEST_CASE(load_addrman_corrupted)
     BOOST_CHECK_THROW(ReadFromStream(addrman2, ssPeers2), std::ios_base::failure);
 }
 
+BOOST_AUTO_TEST_CASE(addrman_update_address)
+{
+    // Tests updating nTime via Connected() and nServices via SetServices()
+    auto addrman = std::make_unique<AddrMan>(std::vector<bool>(), /*deterministic=*/true, /*consistency_check_ratio=*/100);
+    CNetAddr source{ResolveIP("252.2.2.2")};
+    CAddress addr{CAddress(ResolveService("250.1.1.1", 8333), NODE_NONE)};
+
+    int64_t startTime{GetAdjustedTime() - 10000};
```

Comment:
Prefer snake_case for naming local variables in new code:

```suggestion
    int64_t start_time{GetAdjustedTime() - 10000};
```